src/etc/inc/shaper.inc

```diff
@@ -3824,11 +3824,7 @@ class dnpipe_class extends dummynet_class {
 		
 		/* Limiter patch */
 		$selectedScheduler = getSchedulers()[$data['sched']];
-		if (!$selectedScheduler)
-			$input_errors[] = gettext("Selected scheduler not recognized: " . $data['sched'] . '.');
```

Comment:
This should still be tested -- Keep the test but do not put the variable in the error message.